### DISCUSSION TITLE
braid for linux

### DISCUSSION THREAD

**Discussion by iron-lambda** [Gamuxorg/bbs#221] [投递和请求游戏]
求一个linux版的braid（时空幻境），旧版本的在linux上运行会出现各种问题，希望有最新版的道友能够分享一下～

**Comment by shouhuanxiaoji** [comment]
服务器已经打包了。。只是我忘记添加到网站。随后添加一下

**Comment by shouhuanxiaoji** [comment]
https://www.linuxgame.cn/braid 已添加

**Comment by alextome930** [comment]
不知为什么，我无法运行这份appimage，错误显示 /tmp/.mount_1ZKUUs/usr/bin/steam-runtime/run.sh: line 20: /tmp/.mount_1ZKUUs/usr/bin/Braid.bin.x86: No such file or directory。有跟我一样的情况吗？

**Comment by shouhuanxiaoji** [comment]
报一下系统环境和你运行的过程
